test-support/src/main/java/org/springframework/ldap/test/EmbeddedLdapServer.java

```diff
@@ -88,4 +107,49 @@ public void shutdown() throws Exception {
 
         FileUtils.deleteDirectory(workingDirectory);
     }
+
```

Comment:
The changes below seem fairly drastic for updating the apache ds version. Is there a reason they were necessary?